core/templates/dev/head/galleries/gallery.html

```diff
@@ -21,7 +21,12 @@
   <script type="text/javascript">
     GLOBALS.LANGUAGE_CODES_AND_NAMES = JSON.parse(
       '{{LANGUAGE_CODES_AND_NAMES|js_string}}');
-    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse('{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse(
+      '{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+    {% if explorations_list %}
```

Comment:
Actually let's call this ActivitySearchResultsPage and ActivitySearchResultsHandler -- I think that's a better description than Gallery.